man/systemd-analyze.xml

```diff
@@ -1183,6 +1189,14 @@ boot  efi  exitrd  init  lib32	libx32	media	    opt  root  sbin  sys  usr  vmlin
 
       <xi:include href="version-info.xml" xpointer="v258"/>
     </refsect2>
+
+    <refsect2>
+      <title><command>systemd-analyze unit-gdb-attach <replaceable>SERVICE</replaceable></command></title>
```

Comment:
i think we can shorten this to "unit-gdb", it's not that we'd do anything else here than attach